.github/workflows/build.yml

```diff
@@ -122,6 +117,14 @@ jobs:
           LDAP_PASSWORDS: "password1,password2,password3"
     steps:
       - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
+      - name: Install setup-terraform dependencies
+        run: apt-get -y update && apt-get install unzip
```

Comment:
setup-terraform expects unzip to be available but we are running in a docker container that doesn't have it pre-installed